radio/src/targets/common/arm/stm32/usb_driver.cpp

```diff
@@ -46,7 +55,7 @@ int usbPlugged(void)
 
 USB_OTG_CORE_HANDLE USB_OTG_dev;
 
-void OTG_FS_IRQHandler(void)
+extern "C" void OTG_FS_IRQHandler(void)
```

Comment:
This extern "C" should not be necessary if the header already has a extern "C" definition. You can also replace (void)  with () argument in C++.